tests/commands/test_store_commands.py

```diff
@@ -1371,22 +1371,65 @@ def test_revisions_errors_multiple(emitter, store_mock, config, formatted):
 # -- tests for the release command
 
 
-def test_release_simple_ok(emitter, store_mock, config):
-    """Simple case of releasing a revision ok."""
+def test_release_charm_simple_ok(emitter, store_mock, config):
+    """Simple case of releasing a charm revision ok."""
+    store_mock.list_registered_names.return_value = [
+        Entity(
+            name="testcharm",
+            private=False,
+            status="status",
+            entity_type="charm",
+            publisher_display_name="",
+        )
+    ]
     channels = ["somechannel"]
     args = Namespace(name="testcharm", revision=7, channel=channels, resource=[])
     ReleaseCommand(config).run(args)
 
     assert store_mock.mock_calls == [
+        call.list_registered_names(include_collaborations=True),
         call.release("testcharm", 7, channels, []),
     ]
 
     expected = "Revision 7 of charm 'testcharm' released to somechannel"
     emitter.assert_message(expected)
 
 
+def test_release_bundle_simple_ok(emitter, store_mock, config):
+    """Simple case of releasing a bundle revision ok."""
+    store_mock.list_registered_names.return_value = [
+        Entity(
+            name="testbundle",
+            private=False,
+            status="status",
+            entity_type="bundle",
+            publisher_display_name="",
+        )
+    ]
+    channels = ["somechannel"]
+    args = Namespace(name="testbundle", revision=8, channel=channels, resource=[])
+    ReleaseCommand(config).run(args)
+
+    assert store_mock.mock_calls == [
+        call.list_registered_names(include_collaborations=True),
+        call.release("testbundle", 8, channels, []),
+    ]
+
+    expected = "Revision 8 of bundle 'testbundle' released to somechannel"
```

Comment:
Revision 8 of 'testbundle' released to somechannel

And request the store through a bug to return the package type in the release response